charts/unspec-service/values.preview.template.yaml

```diff
@@ -36,7 +36,9 @@ java:
         - docmosis-api-key
         - gov-notify-api-key
         - sendgrid-api-key
+        - launch-darkly-sdk-key
   environment:
+    UNSPECIFIED_ENV: preview
```

Comment:
How about calling it simply `ENV`? Or maybe `LAUNCHDARKLY_ENV`, so it could be grouped in config? Also, `${SERVICE_FQDN}` could be used here, so we could point to specific PRs.